src/lang.js

```diff
@@ -27,7 +27,7 @@
     // Default options //
 
     var defaults = {
-        defaultLocale: 'en' /** The default locale if not set. */
+        defaultLocale: 'en', /** The default locale if not set. */
```

Comment:
@nivla don't worry too much, I have already created an issue/task to include JsHint (see #21).